### PR TITLE
3.8.2 backports

### PR DESCRIPTION
**Pull request by AnotherDroog**
This brings master up to date with changes we introduced in alpine 3.9

What do we want to do about the docker volume splitting issues?

### CODE REVIEW

File: make_container.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Creates containers for docker from a running system with relevant docker images
```
Comment (nolim1t) [inline make_container.sh]: Awesome addition @AnotherDroog

File: home/lncm/compose/docker-compose.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ services:
         ipv4_address: 172.16.88.4
 
   nginx:
-    image: nginx:1.14-alpine-perl
+    image: nginx:1.14.2-alpine
```
Comment (nolim1t) [inline home/lncm/compose/docker-compose.yml]: Another official container right?

**Comment by nolim1t** [comment]
This this working on the PI? 

Would love to try this, but I don't have a HDMI cable with me while I'm on the road.

**Comment by nolim1t** [comment]
Nevermind this is 3.8.2.. I'll try it out tommorrow evening.

File: home/lncm/compose/docker-compose.yml (hunk 2 above)
Comment (AnotherDroog) [reply home/lncm/compose/docker-compose.yml]: Yes, this one is 30MB smaller

**Comment by AnotherDroog** [comment]
Yes I've confirmed it's working. Ready to merge into master

**Comment by nolim1t** [comment]
👏 awesome could you wait a few more days. Would like to do my own testing and give feedback. 

But I have no objections with merging this in now

**Comment by AnotherDroog** [comment]
We can wait, keep testing and let this become the next release for the presentation in BKK

**Comment by AnotherDroog** [comment]
> We can probably also change this too to use sesame OR prompt for a password

I think it should just generate `sesame.txt`

**Pull request merged**